### PR TITLE
#166573650 Fix get all exercises endpoint

### PR DESCRIPTION
**Pull request by wekesa931**
#### Steps to reproduce
After installation and the app runs, navigate to the endpoint: `http://127.0.0.1:8000/api/v2/exercise/` and make a get request.
#### Expected
All exercises should be displayed.
#### Actual
The exercises are not displayed as expected.

### CODE REVIEW

**Review by wangonya — COMMENTED** [review]
~You should probably mention in the description that an `api key` is required to test this 😄. You can add the process of getting the `api key` in the **`steps to reproduce`** section.~

My bad. I was doing a `POST` instead of a `GET` 🤦🏽‍♂️

**Comment by jmusila** [comment]
> You should probably mention in the description that an `api key` is required to test this 😄. You can add the process of getting the `api key` in the **`steps to reproduce`** section.

~Yeah, I think @wekesa931  you can do that.~

**Review by wangonya — APPROVED** [review]
Tested. Works as expected 👍🏾

File: wger/exercises/tests/test_exercise_info.py [hunk 1]
```diff
@@ -1,11 +1,71 @@
 from wger.core.tests.api_base_test import (ApiGetTestCase, ApiBaseTestCase)
 from wger.core.tests.base_testcase import WorkoutManagerTestCase
 from wger.exercises.models import Exercise
+from rest_framework.test import APITestCase
+from rest_framework import status
 
 
-class TestExerciseInfo(ApiBaseTestCase, ApiGetTestCase, WorkoutManagerTestCase):
-
+class TestExerciseInfo(
```
Comment (willeswa) [inline wger/exercises/tests/test_exercise_info.py:8]: Could you consider adding docstrings to explain what the test does?

**Review by willeswa — CHANGES_REQUESTED** [review]

File: wger/exercises/tests/test_exercise_info.py (hunk 1 above)
Comment (wekesa931) [reply wger/exercises/tests/test_exercise_info.py:8]: I have added them. I believe now all is well.

**Review by willeswa — APPROVED** [review]

**Review by jmusila — APPROVED** [review]
LGTM

**Pull request merged**